packages/snap/jest.config.js

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
we should remove commented parameters

Reply:
Thats how they have it in their repo https://github.com/MetaMask/snaps-monorepo/blob/main/jest.config.base.js